Makefile

```diff
@@ -1434,7 +1434,7 @@ fi
 ${MAKE} -f ${SRCDIR}/${THIS} ${IMPL:%=-W %.ml} \
         ${BIN}.cmx LOG_OBJ:=yes 2> $$circ
 if test "$$?" = "0"; then echo "done."
-else echo "FAILED."; exit 1; fi
+else echo "FAILED."; rm -f .*.dis; exit 1; fi
```

Comment:
Wouldn't it be better to use `clean_displays` defined above at line 1397 (https://github.com/rinderknecht/OCaml-build/pull/1/commits/1a5a5aad5eac04d23858ce5df4e69a4054e279ef#diff-b67911656ef5d18c4ae36cb6741b7965R1397) instead of having `rm -f .*.dis` everywhere?